docbook/reference/en/en-US/modules/Migration_from_older_versions.xml

```diff
@@ -12,6 +12,41 @@
         <ulink url="http://docs.jboss.org/resteasy/docs/resteasy-upgrade-guide-en-US.pdf">http://docs.jboss.org/resteasy/docs/resteasy-upgrade-guide-en-US.pdf</ulink>
     </para>
 </sect1>
+<sect1>
+    <title>Migrating to 4.0.0</title>
+    <para>
+       Version 4.0.0 of Resteasy is the first release to support JAX-RS 2.1 (see

```

Comment:
It would be probably better to avoid the statement on 4.0.0 being the "first release to support JAX-RS 2.1", as we will eventually backport JAX-RS 2.1 to the RESTEasy 3 series and at that point people might be confused with ordering.

Reply:
Ok, I changed the section title to "Migrating from 3.0.x to 4.0.0" and opened the section with "In releases 3.0.x". That same text should work for 3.5.